app/celery/tasks.py

```diff
@@ -72,7 +72,10 @@
     SMS_TYPE,
     DailySortedLetter,
 )
-from app.notifications.process_notifications import persist_notification
+from app.notifications.process_notifications import (
+    persist_notification,
+    send_notification_to_queue,
```

Comment:
Refactored this file to use the helper method `send_notification_to_queue`. If we change the logic later, we'll not have to hunt down the logic here, which was the same than in the helper